src/main/webapp/app/exercises/text/assess/text-assessment.component.ts

```diff
@@ -504,4 +504,19 @@ export class TextAssessmentComponent implements OnInit, OnDestroy, AfterViewInit
         const jhiAlert = this.jhiAlertService.error(errorMessage);
         jhiAlert.msg = errorMessage;
     }
+    allowDrop(ev: any) {
+        ev.preventDefault();
+    }
+    drop(ev: any) {
```

Comment:
You might want to refactor the duplicated code for `allowDrop()` and `drop()` **(3)**.